### PR TITLE
Bump vscoq

### PR DESCRIPTION
**Pull request by rtetley**
Replaces #18440

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -36,8 +36,8 @@ variables:
   # The $hash is the first 10 characters of the md5 of the Dockerfile. e.g.
   # echo $(md5sum dev/ci/docker/old_ubuntu_lts/Dockerfile | head -c 10)
   # echo $(md5sum dev/ci/docker/edge_ubuntu/Dockerfile | head -c 10)
-  BASE_CACHEKEY: "old_ubuntu_lts-V2024-01-08-011994e15c"
-  EDGE_CACHEKEY: "edge_ubuntu-V2024-01-08-6b5ac95a11"
+  BASE_CACHEKEY: "old_ubuntu_lts-V2023-11-30-7323af070e"
+  EDGE_CACHEKEY: "edge_ubuntu-V2023-12-29-20017bf137"
```
Comment (SkySkimmer) [inline .gitlab-ci.yml:40]: maybe worth updating the dates here
and why is the base cachekey changing, you only touched the edge dockerfile?

Comment (gares) [inline .gitlab-ci.yml:40]:
```suggestion
  BASE_CACHEKEY: "old_ubuntu_lts-V2024-01-08-011994e15c"
  EDGE_CACHEKEY: "edge_ubuntu-V2024-02-08-20017bf137"
```

**Comment by gares** [comment]
@coqbot run full ci

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -37,7 +37,7 @@ variables:
   # echo $(md5sum dev/ci/docker/old_ubuntu_lts/Dockerfile | head -c 10)
   # echo $(md5sum dev/ci/docker/edge_ubuntu/Dockerfile | head -c 10)
   BASE_CACHEKEY: "old_ubuntu_lts-V2024-01-08-011994e15c"
-  EDGE_CACHEKEY: "edge_ubuntu-V2024-01-08-6b5ac95a11"
+  EDGE_CACHEKEY: "edge_ubuntu-V2024-02-08-20017bf137"
   BASE_IMAGE: "$CI_REGISTRY_IMAGE:$BASE_CACHEKEY"
```
Comment (gares) [inline .gitlab-ci.yml:41]:
```suggestion
  EDGE_CACHEKEY: "edge_ubuntu-V2024-02-0893cae959db"
  BASE_IMAGE: "$CI_REGISTRY_IMAGE:$BASE_CACHEKEY"
```

**Comment by gares** [comment]
@coqbot run full ci

**Comment by rtetley** [comment]
@coqbot run full ci

**Comment by gares** [comment]
@coqbot run full ci

**Comment by gares** [comment]
@rtetley you are now in that group

**Comment by rtetley** [comment]
> @rtetley you are now in that group

:godmode:

**Review by SkySkimmer — APPROVED** [review]

**Comment by SkySkimmer** [comment]
@coqbot merge now

**Pull request merged**